### PR TITLE
Add AddThis (T1596)

### PR DESCRIPTION
**Pull request by MacFan4000**

### CODE REVIEW

File: LocalExtensions.php [hunk 1]
```diff
@@ -5,6 +5,10 @@
 //	require_once( "$IP/extensions/AccessControl/AccessControl.php" );
 //}
 
+if ( $wmgUseAddThis ) {
+	require_once "$IP/extensions/AddThis/AddThis.php";
```
Comment (JohnFLewis) [inline LocalExtensions.php]: consistency is really useful with it enclosed in () as all the ones below.

File: extension-list [hunk 2]
```diff
@@ -1,4 +1,5 @@
 $IP/extensions/AJAXPoll/extension.json
+$IP/extensions/AddThis/extension.json
```
Comment (JohnFLewis) [inline extension-list]: If this exists, why is it being loaded by using "require_once"?

File: LocalExtensions.php (hunk 1 above)
Comment (MacFan4000) [reply LocalExtensions.php]: ?

Comment (JohnFLewis) [reply LocalExtensions.php]: require_once( "$IP/extensions/AddThis/AddThis.php" );

**Comment by MacFan4000** [comment]
Oops wasn’t paying attention to that. Fixed it.

**Pull request merged**